roles/metrics/README.md

```diff
@@ -180,6 +179,20 @@ kubectl describe service prometheus-server -n dojot-monitoring
 
 * After that, just open an ssh tunnel with the endpoint obtained, for example:
 
+## How to deploy monitoring solution?
+
+Initially, it is necessary to instantiate the volumes that will be used by the services to store and obtain log data, metrics and others. Given that, run the command below to instantiate the volumes:
```

Comment:
Eu trocaria o `Given that` por `So`.

Acho que poderia trocar também `instantiate` por `deploy`